duckduckgo/macos-browser
# Convert IconOverlays to Shadow DOM

Task/Issue URL: https://app.asana.com/0/970019368605465/1202944376363831
Tech Design URL:
CC:

**Description**:
Converts the Icon Overlays on YouTube.com into Shadow DOM



**Steps to test this PR**:
1. Make sure `<ddg-icon-overlay>`s are outputted on YouTube.com and works as expected:
   1.  On the homepage (video preview)
   2. In search results
   3. In recommended videos
   4. On user / channel pages
   5. In playlists
6. Run `npx playwright test` (once the integration tests has been merged)

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
